src/proteus/dswx_hls.py

```diff
@@ -4382,21 +4425,21 @@ def _check_ancillary_inputs(check_ancillary_inputs_coverage,
             logger.info(f'    left side (-180 -> +180): {check_1_str}')
 
             # Right side of the antimeridian crossing: +180 -> +360
-            buffer_in_degrees = 0.0002777  # buffer of 1 arcsec: ~ 30m
+            buffer_in_degrees = 33 * 0.0002777  # buffer of 33 arcsec: ~ 1km
```

Comment:
I think it'll be handy to assign `33` to a constant near the top of the file so its easy to modify going forward, just in case we find another tile that needs a slightly larger buffer. Same for the `0.0002777`.

Reply:
Just did it. Thank you, @collinss-jpl